index.html

```diff
@@ -43,6 +43,122 @@ <h4>LET'S CONNECT</h4>
+            <li><a href="#">2015</a></li>
+        </ul>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="lang-names">
+             <li><a href="#">html</a></li>
+            <li><a href="#">css</a></li>
+            <li><a href="#">javascript</a></li>
+        </ul>
+        <button type="button">See Project</button>
+    </div>
+   </div>
+   </div>
+   <div class="container3">
+   <div class="card three">
+    <div class="placeholder-img3"></div>
+    <div class="text">
+        <h3>Tonic</h3>
+        <ul class="tech-dev">
+            <li><a href="#">CANOPY</a></li>
+            <li><a href="#">Back End Dev</a></li>
+            <li><a href="#">2015</a></li>
+        </ul>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="lang-names">
+            <li><a href="#">html</a></li>
+            <li><a href="#">css</a></li>
+            <li><a href="#">javascript</a></li>
+        </ul>
+        <button type="button">See Project</button>
+    </div>
+   </div>
+   </div>
+   <div class="container4">
+   <div class="card four">
+    <div class="placeholder-img4"></div>
+    <div class="text">
+        <h3>Multi-Post Stories</h3>
+        <ul class="tech-dev">
+             <li><a href="#">CANOPY</a></li>
+            <li><a href="#">Back End Dev</a></li>
+            <li><a href="#">2015</a></li>
+        </ul>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="lang-names">
+             <li><a href="#">html</a></li>
+            <li><a href="#">css</a></li>
+            <li><a href="#">javascript</a></li>
+        </ul>
+        <button type="button">See Project</button>
+    </div>
+   </div>
+   </div>
+   </section>
+   <section class="about">
+    <div class="abt-myself">
+    <div class="about-sec">
+        <h1>About Myself</h1>
+        <p>I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+    </div>
+    <div class="social-media-btn">
+        <h4>LET'S CONNECT</h4>
+        <div class="icons">
+            <ul>
+                <li><a href="https://github.com/fauziyahzeeyah"><i class="fa-brands fa-github"></i></a></li>
+                <li><a href="#"><i class="fa-brands fa-linkedin"></i></a></li>
+                <li><a href="https://medium.com/"><i class="fa-solid fa-m"></i></a></li>
+                <li><a href="https://twitter.com/FauziyahZeeyah?t=5x4tHwdJG62bloZ3QehDcw&s=08"><i class="fa-brands fa-twitter"></i></a></li>
+                <li><a href="https://angel.co/"><i class="fa-brands fa-angellist"></i></a></li>
+            </ul>
+        </div>
+        <button type="button"><a href="#">Get My Resume</a></button>
```

Comment:
- [ OPTIONAL ] Kindly fix the alignment here to look like the figma design since it is not aligned well so kindly make it align the same

- [ OPTIONAL ] I think you forget to add border at the top right to the about section so kindly don't forget to add it here like this
![image](https://user-images.githubusercontent.com/58553711/200140868-904033d5-4a28-4281-af56-3d9f34ba1401.png)


- Your design
![image](https://user-images.githubusercontent.com/58553711/200140766-3ba22931-80d1-4d60-8f1a-b5e85568f164.png)

- Figma design
![image](https://user-images.githubusercontent.com/58553711/200140786-314c2fdb-c68c-406e-b213-467a075061ee.png)